autoscale/src/main/resources/application.yml

```diff
@@ -44,6 +44,8 @@ periscope:
       addr: localhost
       port: 5432
   cloudbreak.url: http://localhost:9091
+  notification:
+    endpoint: http://localhost:3000/notifications
   entitlementCheckEnabled: true
```

Comment:
Defaulting to same naming convention as in CB i.e periscope.notification.endpoint.
This will also be initialized with pod specific environment value in dps-k8s repository as done in CB.